### ISSUE TITLE
Fix markdown editor bugs

### ISSUE THREAD

**Issue by avele** [aelve/guide#284]
- clear block button doesn't work
- 4 spaces before text make it "code-text" but after save its regular text
- quote doesn't work

**Comment by neongreen** [comment]
Here's the current 4-space code behavior. It's better but still somewhat broken.

## How it looks in the editor

<img width=200px src="https://user-images.githubusercontent.com/1523306/58370764-97853200-7f0a-11e9-995a-e7753865b70a.png">

## How it looks when rendered

<img width=200px src="https://user-images.githubusercontent.com/1523306/58370771-b683c400-7f0a-11e9-994c-000a8a36f744.png">

## Data on the server

```
One-line sample:

    Hello world

Two-line code sample:

    This is line one
<tab><tab>And this is line two

Ok.
```

**Comment by avele** [comment]
Fixed in @ #296
